chunky/src/java/se/llbit/chunky/model/JigsawModel.java

```diff
@@ -68,8 +77,38 @@
   }
 
   private final Quad[] quads;
+  private final Texture[] textures;
 
   public JigsawModel(String orientation) {
+    switch (orientation) {
+      case "up":
+        textures = textureNoLock;
+        orientation = "up_north";
+        break;
+      case "down":
+        textures = textureNoLock;
+        orientation = "down_north";
+        break;
+      case "north":
+        textures = texturesLock;
+        orientation = "north_up";
+        break;
+      case "south":
+        textures = texturesLock;
+        orientation = "south_up";
+        break;
+      case "east":
+        textures = texturesLock;
+        orientation = "east_up";
+        break;
+      case "west":
+        textures = texturesLock;
+        orientation = "west_up";
+        break;
+      default:
+        textures = texturesLock;
+        break;
+    }
     quads = orientedQuads[getOrientationIndex(orientation)];
```

Comment:
Oh, that's why… Maybe add up,down,… to getOrientationIndex? And a comment that this changed at some point (there should be one in master). I think 1.16?